source/fuzz/transformation_add_constant_composite.cpp

```diff
@@ -72,6 +72,24 @@ bool TransformationAddConstantComposite::IsApplicable(
       }
       break;
     case SpvOpTypeStruct:
+      // We do not create constants of structs decorated with Block nor
+      // BufferBlock.  The SPIR-V spec does not explicitly disallow this, but it
+      // seems like a strange thing to do, so we disallow it to avoid triggering
+      // low priorty edge case issues related to it.
+      if (!ir_context->get_decoration_mgr()->WhileEachDecoration(
+              composite_type_instruction->result_id(), SpvDecorationBlock,
```

Comment:
You can use the following pattern here:
```cpp
for (auto decoration : {SpvDecorationBlock, SpvDecorationBufferBlock}) {
  ...
}
```

Reply:
Done!  My bet is that the older MSVC compilers will complain about lambda capture though.